langchain4j-jlama/src/main/java/dev/langchain4j/model/jlama/JlamaStreamingChatModel.java

```diff
@@ -68,25 +83,74 @@ public static JlamaStreamingChatModelBuilder builder() {
 
     @Override
     public void generate(List<ChatMessage> messages, StreamingResponseHandler<AiMessage> handler) {
+        generate(messages, List.of(), handler);
+    }
+
+    @Override
+    public void generate(List<ChatMessage> messages, ToolSpecification toolSpecification, StreamingResponseHandler<AiMessage> handler) {
+        generate(messages, List.of(toolSpecification), handler);
```

Comment:
The `generate(List<ChatMessage>, ToolSpecification, StreamingResponseHandler<AiMessage>)` method is meant to **_force_** the LLM to execute the specified tool. It this supported by Jlama? If not, it would be better to throw an unsupported exception

Reply:
BTW, same applies to `JlamaChatModel.generate(List<ChatMessage>, ToolSpecification)`

Reply:
I don't know of any open models that do this, you kind of just ask the LLM.  I guess it could add a system prompt like "You MUST use the tool specified".  Do you want me todo that?

Reply:
I see, then better to just throw an exception that this is not supported

Reply:
Ok.. Can you should fix the javadoc on this method as there's no indication of this invariant.

Reply:
Oh the javadoc is correct on `ChatLanguageModel` but not `StreamingChatLanguageModel`

Reply:
Oh good catch, thanks! I will update it